src/widgets/splits/SplitHeader.cpp

```diff
@@ -670,6 +680,9 @@ void SplitHeader::enterEvent(QEvent *event)
 {
     if (!this->tooltipText_.isEmpty())
     {
+        dynamic_cast<TwitchChannel *>(this->split_->getChannel().get())
+            ->refreshTitle();
```

Comment:
Check if it is actually a twitch channel (what if we add mixer and it also has titles).